assets/dns/daemonset.yaml

```diff
@@ -2,6 +2,8 @@ kind: DaemonSet
 apiVersion: apps/v1
 # name, namespace and labels are set at runtime
 spec:
+  # minReadySeconds should be 3x the readiness probe's polling interval.
```

Comment:
```suggestion
  # minReadySeconds should be 3x the readiness probe's polling interval (periodSec)
```

Reply:
Updated in https://github.com/openshift/cluster-dns-operator/compare/b0ca398b84de1287fe02d22bf9311f0053a02d1c..043594d697605804dae0952b77c741270c0042bc.